NxMap.lua

```diff
@@ -678,7 +678,7 @@ function Nx.Map:Create (index)
 	f:SetFrameStrata ("LOW")
 	f:SetWidth (m.W)
 	f:SetHeight (m.H)
-	f:SetMinResize (50, 50)
+	f:SetResizeBounds (50, 50)
```

Comment:
`SetMinResize` has been removed and replaced by `SetResizeBounds`, see https://wowpedia.fandom.com/wiki/API_Frame_SetMinResize